src/amo/components/CollectionManager/styles.scss

```diff
@@ -17,6 +17,7 @@
   input,
   textarea {
     width: 100%;
+    resize: vertical;
```

Comment:
Hey great work, could you please change the order of `resize` and `width` so that it will be in alphabetical order.

Reply:
Okay, will do that right away!
Thanks for the helpful link !!